doc/lz4_Frame_format.md

```diff
@@ -85,14 +85,13 @@ That’s where compressed data is stored.
 
 __EndMark__
 
-The flow of blocks ends when the last data block has a size of “0”.
-The size is expressed as a 32-bits value.
+The flow of blocks ends when the last data block is announced with
+an invalid size of “0”, expressed as a 32-bits value (exactly `0x00000000`).
```

Comment:
Maybe reference the `__Block Size__` section, because I don't know what an "invalid size of 0" means until I read that section.

Reply:
Wording has been simplified, to only mention a "value of `0x00000000`",
thus making the concept of "size" irrelevant in the context of `endMark`.